connectors/cli.py

```diff
@@ -152,6 +173,11 @@ def run(args):
 
 
 def main(args=None):
+    """Main entry point.
+
+    Parses the arguments and calls `run` with them.
+    If `--version` is used, it displays the version and exits.
+    """
```

Comment:
should this description focus more on usage of `main` function? 

Something along the lines with

"This function is the entry point to the Connectors Service, it's responsible for all operations that the service can do:

- Listing all connectors (when args.action == 'list')
- Running connector service with provided args and configuration file stored in config.yaml
"

Reply:
sounds good!